app/src/main/java/com/example/myhouse_aos/data/model/request/RequestAddFolderDto.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.myhouse_aos.data.model.request
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class RequestAddFolderDto(
+    @SerialName("image_url")
+    val image_url: String?,
```

Comment:
코틀린에서 사용하는 변수는 다 camleCase 로 바꿔주세용~
```suggestion
    @SerialName("image_url")
    val imageUrl: String?,
```